assets/i18n/qqq.json

```diff
@@ -575,7 +576,23 @@
 	"I18N_TOPNAV_SIGN_IN_WITH_GOOGLE": "Text displayed inside the Sign In dropdown. - When clicked, the user is signed in with Google Oauth - The text in the button needs to be less than 25 characters long",
 	"I18N_TOPNAV_TEACH_WITH_OPPIA": "Text displayed inside drop-down menu - The list is shown after the user hovers over the I18N_TOPNAV_ABOUT button. When the option is clicked, the teach with oppia page is loaded which contains information specific for teachers who would like to create explorations.",
 	"I18N_TOPNAV_TOPICS_AND_SKILLS_DASHBOARD": "Text displayed in the dropdown on the navbar for reaching the topics and skills dashboard.",
+	"I18N_TOPNAV_WELCOME_GUIDE_PAGE": "Welcome Guide",
 	"I18N_TOTAL_SUBSCRIBERS_TEXT": "Text displayed under the subscribers tab in creator dashboard. Tells the creator the total number of subscribers he/she has.",
 	"I18N_UNSUBSCRIBE_BUTTON_TEXT": "The text that appears on the unsubscribe button, which allows users to unsubscribe from creators.\n{{Identical|Unsubscribe}}",
+    "I18N_WELCOME_GUIDE_TITLE": "Teachers and Parents: A Guide to Oppia",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_1": "Search bar used to search for a specific exploration",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_2": "Category Filter to filter explorations by topic (music, math, biology, etc)",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_3": "Language Filter to filter explorations by language",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_4": "Navigation button that you can click on to browse through more explorations",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_5": "Language Selector used to change the language of the entire page",
+    "I18N_WELCOME_PAGE_COLLECTIONS_PARAGRAPH": "A Collection in Oppia is a library of lessons.  Oppia has several collections, including the “Fractions” collection. Within the “Fractions” collection, there are different explorations that help a student strengthen their fraction skills. ",
+    "I18N_WELCOME_PAGE_COLLECTIONS_TITLE": "What is a Collection?",
+    "I18N_WELCOME_PAGE_EXPLORATION_PARAGRAPH": "At Oppia, the lessons we provide for students are called explorations. All Explorations are comprised of images, videos, and questions that help students explore and strengthen their knowledge on various topics in a fun and interactive way. The first exploration in the Fraction collection is called “What is a Fraction?” ",
+    "I18N_WELCOME_PAGE_EXPLORATION_TITLE": "What is an Exploration?",
+    "I18N_WELCOME_GUIDE_HOW_TO_TITLE": "How Can I Find an Explorations and Collections?",
```

Comment:
an is singular right? Explorations -> Exploration

Reply:
Done!